src/components/ContributorShowcase/ContributorShowcase.module.css

```
@@ -0,0 +1,41 @@
+@media (min-width: 300px) { 
+  .grid {
+    display: grid;
+    grid-template-columns: auto auto;
+  }
+}
+
+
+@media (min-width: 768px) { 
+  .grid {
+    display: grid;
+    grid-template-columns: auto auto auto auto;
+  }
+}
+
+@media (min-width: 992px) { 
+  .grid {
+    display: grid;
+    grid-template-columns: auto auto auto auto auto auto auto;
+  }
+}
+
+@media (min-width: 1200px) { 
+  .grid {
+    display: grid;
+    grid-template-columns: auto auto auto auto auto auto auto;
+  }
+}
+
+.image {
+  min-width: 100px;
+  height: 100px;
+  margin-left: auto;
+  margin-right: auto;
+  display: block;
+}
+
+.griditem {
+  padding: 10px;
+  @apply text-xs text-center;
+}
```

should end on a newline
```suggestion
}

```